server/services/voip/settings.ts

```diff
@@ -0,0 +1,93 @@
+import { settingsRegistry } from '../../../app/settings/server';
```

Comment:
Can you please add these settings on "app/lib/server/startup/settings.ts" file? 👀